packages/hardhat/package.json

```diff
@@ -6,7 +6,7 @@
     "node": "12.x"
   },
   "scripts": {
-    "test": "hardhat test",
+    "test": "hardhat test --network hardhat",
```

Comment:
Will your tests work on the alfajores testnet? I think the current test scripts run on alfajores by default.

Reply:
Yep, just tested :)

Reply:
I reverted  the `test` script in package.json and added a new one `test-local`. Please let me know if you would prefer to not add it or if you want me to add any other changes to this Pull Request